[2022-07-26 09:31:56,036] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v1.load_to_snowflake manual__2022-07-26T09:31:20.211065+00:00 [queued]>
[2022-07-26 09:31:56,052] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v1.load_to_snowflake manual__2022-07-26T09:31:20.211065+00:00 [queued]>
[2022-07-26 09:31:56,052] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-26 09:31:56,052] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-26 09:31:56,053] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-26 09:31:56,087] {taskinstance.py:1397} INFO - Executing <Task(S3ToSnowflakeOperator): load_to_snowflake> on 2022-07-26 09:31:20.211065+00:00
[2022-07-26 09:31:56,106] {standard_task_runner.py:52} INFO - Started process 1571 to run task
[2022-07-26 09:31:56,114] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dataquality_pipeline_v1', 'load_to_snowflake', 'manual__2022-07-26T09:31:20.211065+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/dataquality_pipeline_v1.py', '--cfg-path', '/tmp/tmpai73fsr0', '--error-file', '/tmp/tmpspdakygw']
[2022-07-26 09:31:56,125] {standard_task_runner.py:80} INFO - Job 112: Subtask load_to_snowflake
[2022-07-26 09:31:56,313] {task_command.py:371} INFO - Running <TaskInstance: dataquality_pipeline_v1.load_to_snowflake manual__2022-07-26T09:31:20.211065+00:00 [running]> on host d19767594dfa
[2022-07-26 09:31:56,520] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dataquality_pipeline_v1
AIRFLOW_CTX_TASK_ID=load_to_snowflake
AIRFLOW_CTX_EXECUTION_DATE=2022-07-26T09:31:20.211065+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-26T09:31:20.211065+00:00
[2022-07-26 09:31:56,522] {s3_to_snowflake.py:128} INFO - Executing COPY command...
[2022-07-26 09:31:56,548] {base.py:68} INFO - Using connection ID 'snowflake_default' for task execution.
[2022-07-26 09:31:56,559] {connection.py:262} INFO - Snowflake Connector for Python Version: 2.7.9, Python Version: 3.7.13, Platform: Linux-5.15.0-1014-azure-x86_64-with-debian-11.4
[2022-07-26 09:31:56,560] {connection.py:877} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2022-07-26 09:31:56,561] {connection.py:894} INFO - Setting use_openssl_only mode to False
[2022-07-26 09:31:58,390] {cursor.py:710} INFO - query: [ALTER SESSION SET autocommit=True]
[2022-07-26 09:31:58,663] {cursor.py:734} INFO - query execution done
[2022-07-26 09:31:58,664] {snowflake.py:328} INFO - Running statement: COPY INTO YELLOW_TRIPDATA
FROM @YELLOW_TRIPDATA_STAGE/
file_format=(type = 'CSV', skip_header = 1, time_format = 'YYYY-MM-DD HH24:MI:SS'), parameters: None
[2022-07-26 09:31:58,664] {cursor.py:710} INFO - query: [COPY INTO YELLOW_TRIPDATA FROM @YELLOW_TRIPDATA_STAGE/ file_format=(type = 'CSV'...]
[2022-07-26 09:32:05,025] {cursor.py:734} INFO - query execution done
[2022-07-26 09:32:05,025] {snowflake.py:338} INFO - Statement execution info - {'file': 's3://snowflakedata212/yellow_tripdata_sample_2019-01.csv', 'status': 'LOADED', 'rows_parsed': 10000, 'rows_loaded': 10000, 'error_limit': 1, 'errors_seen': 0, 'first_error': None, 'first_error_line': None, 'first_error_character': None, 'first_error_column_name': None}
[2022-07-26 09:32:05,026] {snowflake.py:338} INFO - Statement execution info - {'file': 's3://snowflakedata212/yellow_tripdata_sample_2019-02.csv', 'status': 'LOADED', 'rows_parsed': 10000, 'rows_loaded': 10000, 'error_limit': 1, 'errors_seen': 0, 'first_error': None, 'first_error_line': None, 'first_error_character': None, 'first_error_column_name': None}
[2022-07-26 09:32:05,026] {snowflake.py:342} INFO - Rows affected: 2
[2022-07-26 09:32:05,026] {snowflake.py:343} INFO - Snowflake query id: 01a5dc9b-0000-da87-0000-22590005d51a
[2022-07-26 09:32:05,026] {connection.py:507} INFO - closed
[2022-07-26 09:32:05,259] {connection.py:510} INFO - No async queries seem to be running, deleting session
[2022-07-26 09:32:05,518] {s3_to_snowflake.py:130} INFO - COPY command completed
[2022-07-26 09:32:05,547] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dataquality_pipeline_v1, task_id=load_to_snowflake, execution_date=20220726T093120, start_date=20220726T093156, end_date=20220726T093205
[2022-07-26 09:32:05,607] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-26 09:32:05,761] {local_task_job.py:273} INFO - 8 downstream tasks scheduled from follow-on schedule check
